README.md

```diff
@@ -152,18 +115,6 @@ my $vars = {
 };
 ```
 
-### Prefer vertically aligned assignments
```

Comment:
Assignments are not always vertically aligned in our code base. I don't think this should be a hard rule. IMO, I think it should be an "at your discretion" thing.